adot-testbed/app/src/test/java/software/amazon/adot/testbed/LogsTests.java

```diff
@@ -0,0 +1,145 @@
+package software.amazon.adot.testbed;
+import java.io.FileWriter;
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+
+import com.github.rholder.retry.RetryerBuilder;
+import com.github.rholder.retry.StopStrategies;
+import com.github.rholder.retry.WaitStrategies;
+import net.bytebuddy.asm.Advice;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.MountableFile;
+import software.amazon.awssdk.services.cloudwatchlogs.CloudWatchLogsClient;
+import software.amazon.awssdk.services.cloudwatchlogs.model.GetLogEventsRequest;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.InputStreamReader;
+import java.io.InputStream;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@Testcontainers(disabledWithoutDocker = true)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class LogsTests {
+    private static final String LOCAL_CREDENTIALS = System.getProperty("adot.testbed.localcreds");
```

Comment:
What are the local credentials for?

Reply:
We can use the system property to provide credentials from the local terminal, the command would be something like `./gradlew -Dadot.testbed.localcreds="~/.aws" test --rerun-tasks --info`.  [at this point](https://github.com/aws-observability/aws-otel-test-framework/pull/1426/files#diff-b6b213810cf9bd4029972a83c70ccf32f71e4f4eb1e32cc5b6b90dd5bcf7961aR77) it would mount like `~/.aws:/root/.aws`.